packages/py3_atspi.rb

```diff
@@ -5,7 +5,7 @@
 
 class Py3_atspi < Meson
   description 'Python bindings for D-Bus AT-SPI'
-  homepage 'https://wiki.linuxfoundation.org/accessibility/atk/at-spi/at-spi_on_d-bus'
+  homepage 'http://www.linuxfoundation.org/en/AT-SPI_on_D-Bus'
```

Comment:
This url is not found.  The old url is working.  Why the change?

Reply:
For anitya, but the old one works fine.